distribution/test/itests/test-itests-dependencies-app/src/main/filtered-resources/features.xml

```diff
@@ -38,13 +38,13 @@
     </feature>
 
     <feature name="third-party-dependencies" version="${project.version}">
-        <bundle>mvn:commons-io/commons-io/2.5</bundle>
+        <bundle>mvn:commons-io/commons-io/${commons-io.version}</bundle>
```

Comment:
Nice using the property for versions, could make maintenance a little easier down the road.